sync/class.jetpack-sync-client.php

```diff
@@ -343,20 +355,12 @@ function send_theme_info() {
 		 * @param object the theme support hash
 		 */
 		do_action( 'jetpack_sync_current_theme_support', $theme_support );
+		return 1;
```

Comment:
Why do we return 1 here? And should it be an integer as opposed to a string or a boolean?